pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -186,14 +187,19 @@ class SystemInformationSshKeys extends React.Component {
         else if (!this.state.keys.length)
             body = <EmptyStatePanel title={ _("No host keys found.") } />;
         else
-            body = <div className="list-group dialog-list-ct">
-                {this.state.keys.map(key =>
-                    <div className="list-group-item" key={key.title}>
-                        <h4>{key.title}</h4>
-                        {key.fps.map((fp, i) => <div key={i}><small>{fp}</small></div>)}
-                    </div>
-                )}
-            </div>;
+            body = <ListingTable aria-label={_("SSH key fingerprints")}
+                    variant="compact"
+                    columns={[""]}
+                    rows={this.state.keys.map(key => {
+                        return {
```

Comment:
If you have to force-push for some reason (pixel tests?), please `k => { return v}` → `k => v`. (Otherwise, not important enough to re-run everything_

Reply:
I did this because I need to return object. I guess there is way how to do it, but I always used `return` when I was returning object. Any help with the syntax as doing `k => {foo: "bar"}` does not work.

Reply:
`k => ({ foo: "bar" })` works. Similar to destructuring in arrow arguments.